Home_work6.py

```diff
@@ -0,0 +1,52 @@
+import random
+#Task 1.1 не запускати, бо майже infinity loop, з меньшими цифрами працює)
+a = 1000000000
+b = 9999999999
+some_random = random.randint(a, b)
+while some_random != b:
+    some_random = random.randint(a, b)
+    if some_random  == b:
+        print(some_random)
+        break
+
+#Task 1.2 # Ось відповідь на першу задачу
+a = 1000000000
+b = 9999999999
+count = 0
+some_random_list = list()
+while count < 10:
+    some_random = random.randint(a, b)
+    some_random_list.append(some_random)
+    count += 1
+    if count == 10:
+        print (max(some_random_list)) 

```

Comment:
you dont need numbers with 10 digits in it, you need list with 10 random numbers